### PR TITLE
Mac: implement DropDownToolItemHandler

### PR DESCRIPTION
**Pull request by hazzik**
[!Screenshot](https://user-images.githubusercontent.com/144791/173305871-f26a063f-61c8-44f9-a084-a3bfa9992d2e.mov)

### CODE REVIEW

File: projects/GKv3/GEDKeeper3/GKUI/Platform/DropDownToolItem.cs [hunk 1]
```diff
@@ -184,18 +185,25 @@ public ContextMenu ContextMenu
...
 
-    public class ButtonToolItemHandler : ToolItemHandler<NSToolbarItem, ButtonToolItem>, ButtonToolItem.IHandler
+    public class DropDownToolItemHandler : ToolItemHandler<NSToolbarItem, DropDownToolItem>, DropDownToolItem.IHandler
     {
         ContextMenu contextMenu;
 
+        public new string Text
+        {
+            get => Button.Title;
+            set => Button.Title = value;
```
Comment (hazzik) [inline projects/GKv3/GEDKeeper3/GKUI/Platform/DropDownToolItem.cs:195]: By default text is set to a label underneath the button. We want it to be in the button itself.

File: projects/GKv3/GEDKeeper3/GKUI/Platform/DropDownToolItem.cs [hunk 2]
```diff
@@ -184,18 +185,25 @@ public ContextMenu ContextMenu
...
 
-    public class ButtonToolItemHandler : ToolItemHandler<NSToolbarItem, ButtonToolItem>, ButtonToolItem.IHandler
+    public class DropDownToolItemHandler : ToolItemHandler<NSToolbarItem, DropDownToolItem>, DropDownToolItem.IHandler
     {
         ContextMenu contextMenu;
 
+        public new string Text
+        {
+            get { return Button.Title; }
+            set { Button.Title = value; }
```
Comment (hazzik) [inline projects/GKv3/GEDKeeper3/GKUI/Platform/DropDownToolItem.cs:195]: By default text is set to a label underneath the button. We want it to be in the button itself.

Comment (hazzik) [reply projects/GKv3/GEDKeeper3/GKUI/Platform/DropDownToolItem.cs:195]: Shall we change it to only if image is null?

**Review by Serg-Norseman — APPROVED** [review]
👍

**Pull request merged**